src/com/lftechnology/java/training/dipak/employeemanagement/ui/MainClass.java

```diff
@@ -0,0 +1,69 @@
+
+package com.lftechnology.java.training.dipak.employeemanagement.ui;
+
+import java.util.Scanner;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.dipak.employeemanagement.LoggerFormatter;
+import com.lftechnology.java.training.dipak.employeemanagement.controller.LoginController;
+import com.lftechnology.java.training.dipak.employeemanagement.domain.Employee;
+import com.lftechnology.java.training.dipak.employeemanagement.domain.User;
+import com.lftechnology.java.training.dipak.employeemanagement.domain.UserType;
+
+/**
+ * <p>
+ * This class contains a main method. Hence the execution takes place from that method.
+ * </p>
+ * 
+ * @author Dipak Thapa<dipakthapa@lftechnology.com>
+ */
+public class MainClass {
+
+    private static final Logger LOGGER = Logger.getLogger(MainClass.class.getName());
+    public static boolean isRunApplicationAgain = true;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make isRunApplicationAgain a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make this "public static isRunApplicationAgain" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1444)